plugins/inventory/vmware_vm_inventory.py

```diff
@@ -227,17 +244,21 @@ def _login_vapi(self):
         """
         session = requests.Session()
         session.verify = self.validate_certs
-        if not self.validate_certs:
-            # Disable warning shown at stdout
-            requests.packages.urllib3.disable_warnings()
+        # if not self.validate_certs:
```

Comment:
I think we should hide the warnings to stdout.

Reply:
I belief I comment that for debug, but for forget to uncomment it.